UnityProject/Assets/Scripts/Weapons/WeaponNetworkActions.cs

```diff
@@ -152,15 +152,15 @@ public void CmdUnloadWeapon(GameObject gunObject)
 				{
 					SimpleAnimal attackTarget = victim.GetComponent<SimpleAnimal>();
 					RpcMeleeAttackLerp(attackDirection, weapon);
-					playerMove.allowInput = false;
+					//playerMove.allowInput = false;
```

Comment:
I think it's OK to remove those lines. Or you could add a comment to explain why the lines are commented.